package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "channelape-sdk",
-  "version": "1.30.0-develop.2",
+  "version": "1.31.0-develop.0",
```

Comment:
@ryankazokas Can you change to `1.30.0-develop.3` as we'll deploy both the step/supplier changes and this change in the same version (1.30.0)?